components/event-receiver/event-input-adapters/org.wso2.carbon.event.input.adapter.websocket.local/src/main/java/org/wso2/carbon/event/input/adapter/websocket/local/WebsocketLocalEventAdapterFactory.java

```diff
@@ -45,6 +45,11 @@ public String getType() {
     }
 
     @Override
+    public String getUsageTips() {
+        return WebsocketLocalEventAdapterConstants.ADAPTOR_USAGE_TIPS_WEBSOCKET_LOCAL;
```

Comment:
@lasanthafdo 
Yes, we can do that and thank you for pointing it out. Fixed in PR: https://github.com/wso2/carbon-analytics-common/pull/144